winui/Rating/ToolTip.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: ToolTip with WinUI Rating control | Syncfusion
+title: ToolTip in WinUI Rating control | Syncfusion
 description: Learn here all about how to enable the tooltip and format the tooltip content in WinUI Syncfusion Rating (SfRating) control.
```

Comment:
description: Learn all about how to enable the tooltip and format the tooltip content in WinUI Syncfusion Rating (SfRating) control here.